tests/test_settings.py

```diff
@@ -0,0 +1,54 @@
+from importlib import reload
+from django.test import SimpleTestCase
+
+from webhook_receiver import settings as default_settings
+from webhook_receiver.settings import production as production_settings
+
+
+class DefaultSettingsTest(SimpleTestCase):
```

Comment:
So @foadlind the reason that my test result didn't match what you saw in #10 was that I previously used `TestCase` here which does some mangling with the database settings. `SimpleTestCase` doesn't do that and thus reflects the database settings accurately.